deploy/cookbook_artifact_versions.sql

```diff
@@ -0,0 +1,25 @@
+-- Deploy cookbook_artifact_versions
+-- requires: cookbook_artifacts
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS cookbook_artifact_versions(
+  id CHAR(32) PRIMARY KEY,
+  identifier VARCHAR(255) NOT NULL,
+  meta_attributes bytea NOT NULL,
+  meta_long_desc bytea NOT NULL,
+  metadata bytea NOT NULL,
+  serialized_object bytea NOT NULL,
+  updated_at TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+  created_at TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+  last_updated_by CHAR(32) NOT NULL,
+  cookbook_id INTEGER NOT NULL REFERENCES cookbooks(id) ON DELETE RESTRICT,
+  UNIQUE(cookbook_id, identifier)
+);
+
+ALTER TABLE cookbook_artifact_versions ALTER meta_attributes SET STORAGE EXTERNAL;
```

Comment:
I believe EXTERNAL controls whether or not pg does compression on the data (although that's probably not all it means).